Index: electrum-zcash-post-3.2.2/electrum-zcash
===================================================================
--- electrum-zcash-post-3.2.2.orig/electrum-zcash
+++ electrum-zcash-post-3.2.2/electrum-zcash
@@ -27,6 +27,8 @@ import sys
 import sys
 
 
+os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION']='python'
+
 MIN_PYTHON_VERSION = "3.6.1"  # FIXME duplicated from setup.py
 _min_python_version_tuple = tuple(map(int, (MIN_PYTHON_VERSION.split("."))))
 
